discovery-aws-api-async/src/main/scala/akka/discovery/awsapi/ecs/AsyncEcsSimpleServiceDiscovery.scala

```diff
@@ -25,7 +27,10 @@ class AsyncEcsSimpleServiceDiscovery(system: ActorSystem) extends SimpleServiceD
   private[this] val config = system.settings.config.getConfig("akka.discovery.aws-api-ecs-async")
   private[this] val cluster = config.getString("cluster")
 
-  private[this] lazy val ecsClient = ECSAsyncClient.create()
+  private[this] lazy val ecsClient = {
+    val conf = ClientOverrideConfiguration.builder().retryPolicy(RetryPolicy.NONE).build()
```

Comment:
Is there a reason not to do this in the sync version too (EcsSimpleServiceDiscovery.scala)?